src/data/HcvSettingsJsonRepository.ts

```diff
@@ -0,0 +1,9 @@
+import { HcvSettings } from "domain/entities/HcvSettings";
+import { HcvSettingsRepository } from "domain/repositories/HcvSettingsRepository";
+import { readFileSync } from "jsonfile";
+
+export class HcvSettingsJsonRepository implements HcvSettingsRepository {
```

Comment:
As discussed with Jorge, we won't have a repo for this, the script will read the JSON file and pass the object to the use case.